### PR TITLE
 Add twbs-pagination@1.3.1

### PR DESCRIPTION
**Pull request by Piicksarn**
**git repo url:** https://github.com/esimakin/twbs-pagination
**Watch 20
  Star 254
 Fork 93**
@maruilian11,  could you help for me check out the PR, which is referenced to  #6414 ? thank you~
- [x] Not found on cdnjs repo
- [x] No already exist issue and PR
- [x] Notable popularity
- [x] Project has public repository on famous online hosting platform (or been hosted on npm) 
- [x] Has valid tags for each versions (for git auto-update)
- [x] Auto-update setup
- [x] Auto-update target/source is valid.
- [x] Auto-update filemap is correct

### CODE REVIEW

File: ajax/libs/twbs-pagination/package.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  "dependencies": {
+    "jquery": ">=1.7.0"
+  },
+  "keywords": [
+    "pagination",
+    "jQuery",
+    "jQuery-plugin",
+    "bootstrap"
+  ],
+  "license": "Apache 2.0",
```
Comment (PeterDaveHello) [inline ajax/libs/twbs-pagination/package.json]: Wrong style ...

Comment (AngeIII) [inline ajax/libs/twbs-pagination/package.json]: Hi, @PeterDaveHello ! Does this related to https://github.com/esimakin/twbs-pagination ? We can fix this, but what exactly is wrong?

Comment (PeterDaveHello) [inline ajax/libs/twbs-pagination/package.json]: Nope, not really, we use SPDX identifier in cdnjs (#5543).

**Comment by PeterDaveHello** [comment]
We have a small problem here, please rebase with our latest master branch, sorry for the inconvenient caused.

**Comment by Piicksarn** [comment]
@maruilian11, would you mind to have a look at the pr?
Thank you!

**Comment by maruilian11** [comment]
@Piicksarn you miss `repository` and please open a issue to ask the author to add tags to 1.3.1.

**Comment by Piicksarn** [comment]
@maruilian11, there is already a tag for v1.3.1 & i have added the repository config~
would you please have a look at the pr again?
Thank you!

**Comment by Piicksarn** [comment]
@maruilian11, @Amomo,
would you please have a look at the pr?
Thank you!

**Comment by maruilian11** [comment]
@Piicksarn sorry for checking late, could you rebase the commit.again? Thank you!

**Comment by Piicksarn** [comment]
@maruilian11
would you please have a look at the pr?

**Pull request merged**